string ns
string query
---
OntologeniusQueryResponse[] results
string error
